static/templates/homepage.html

```diff
@@ -0,0 +1,262 @@
+<div layout="row">
+    <md-content layout="column" layout-fill>
+        <!-- Main Container for the image-->
+        <div class="hero-area">
+            <!--The toolbar on the image-->
+            <div layout="row" layout-align="end start" ng-controller="LoginController as login">
+                <md-button md-no-ink="" class="menuhp">How it works</md-button>
+                <!--md-button  md-no-ink="" ng-href="/login" class="menuhp">Login</md-button-->
+                <md-button md-no-ink="" class="menuhp" ng-click="login.showLoginForm($event)">Login</md-button>
+                <md-button md-no-ink="" ng-href="/" class="menuflag"><img src="/static/images/homepage/english.png" width="50%" height="auto" alt="" /></md-button>
```

Comment:
Use of external graphics and images can be tricky. We don't know the copyrights of these. We can take it out until we design something on our own